driver/src/main/com/mongodb/Mongo.java

```diff
@@ -44,20 +43,14 @@
 import com.mongodb.operation.ReadOperation;
 import com.mongodb.operation.WriteOperation;
 import com.mongodb.selector.LatencyMinimizingServerSelector;
-import org.bson.BsonBinary;
```

Comment:
Surprised by the number of removals here!

Reply:
Yeah, I'm glad to have the session pool out of here.